### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by Seonlo99**
Update the following commands in User Guide.

- `addaffn`
- `removeah`
- `nok`
- `edit`

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -150,12 +150,12 @@ The `Model` component,
 
 The `Person` component has several attributes: `Name`, `Phone`, `Email`, `Role`, `Affiliations` and `AffiliationHistory`. Each instance of `Person` class has exactly one of each of these attributes. In particular, `Affiliations` and `AffiliationHistory` can have multiple `Affiliation`. 
 
+`Patient` and `Staff` are the 2 main subclasses of `Person`. `Patient` has an additional attribute `NextOfKin`. `Staff` has an additional attribute `ShiftDays`. `Staff` is a superclass of `Doctor` and `Nurse`, where the former has an additional attribute `Specialisation`.
+
```
Comment (EricXiong420) [inline docs/DeveloperGuide.md:154]: Shift in paragraph makes the DG make more sense, good.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -136,6 +137,26 @@ Examples:
...
+* If person at index is staff, affiliations specified must be a patient.
+* If person at index is patient, affiliations specified must be a staff.
+* Adding patient A to affiliation of staff B, will result in adding staff B to affiliation of Patient A.
+* Multiple affiliations can be added.
+* At least one affiliation field must be provided.
+* `AFFN` is specified as full name of a person in the contact list.
+
+Examples:
+* `addaffn 1 a/John` add `John` to affiliation of 1st person in the contact list.
+* `addaffn 3 a/John a/Mary` add `John` and `Mary` to affiliation of 3rd person in the contact list.
```
Comment (EricXiong420) [inline docs/UserGuide.md:158]: Readable and contains just enough details for user, good.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -157,11 +178,24 @@ Finds the staff/patients that used to be affiliated or are currently affiliated
...
-* Deleting a person automatically deletes the person from others' affilation history.
 
 Examples:
 * `affnh 2` lists the people who used to be affiliated or are currently affiliated to the 2nd person in the contact list.
 
+### Remove affiliation history of a person: `removeah`
+
+Format: `removeah INDEX`
+
+Remove all affiliation history of patient/staff indicated by the given `INDEX`, except for affiliations that are currently affiliated.
```
Comment (EricXiong420) [inline docs/UserGuide.md:189]: I haven't seen the first part of the UG but may be good to add a section explaining what's the difference between affiliations and affiliation history (or that we store affiliation history in the first place).

File: docs/UserGuide.md [hunk 4]
```diff
@@ -157,11 +178,24 @@ Finds the staff/patients that used to be affiliated or are currently affiliated
...
+
+* Remove all affiliation history for the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index **must be a positive integer** 1, 2, 3, …​
+* Current affiliation in affiliation history will not be removed.
```
Comment (EricXiong420) [inline docs/UserGuide.md]: Maybe something like "Does not remove the person they are currently affiliated to from affiliation history"

**Review by EricXiong420 — COMMENTED** [review]
Minor change requested, otherwise looks good!

File: docs/UserGuide.md (hunk 3 above)
Comment (Seonlo99) [reply docs/UserGuide.md:189]: Yea I think it is a good idea, we can discuss about it later in class.

File: docs/UserGuide.md (hunk 4 above)
Comment (Seonlo99) [reply docs/UserGuide.md]: Right! Thats sounds better will change it in the next commit!

**Review by leezhanpeng — APPROVED** [review]
LGTM. Will merge it, and if there are any changes required, can just follow up with another PR.

**Pull request merged**